### DISCUSSION TITLE
How To Add a Colour Property To Script

### DISCUSSION THREAD

**Discussion by Epster-18** [derekantrican/GAS-ICS-Sync#361] [Q&A] [answered]
Hi Derek and team - Thanks so much for the incredible script.

Could someone please show me how to set a specific colour for the events that are being synced into my Google Calendar. I was unfortunately forced to add the sync to my "main" work calendar and due to that all these synced events are the same colour as other events I have in my main calendar.

If someone could show me where in the script I need to add what text to set a specific colour for the events that get sent over that would truly make my day. I don't have a development background and don't want to break anything (eyesclosed).

If I haven't explained myself well please let me know.

Thanks so much!

**Comment by derekantrican** [comment, marked as answer]
This should be in some of the helper text here: https://github.com/derekantrican/GAS-ICS-Sync/blob/master/Code.gs#L27

You can specify a calendar with color like this: `["https://p24-calendars.icloud.com/holidays/us_en.ics", "US Holidays", "11"]` (where the last value is a color id from here: https://developers.google.com/apps-script/reference/calendar/event-color

**Reply by Epster-18** [reply]
Unbelievable! You have no idea how much I've been struggling with this and only saw this response now.
To think it was so simple this entire time! Thanks so much @derekantrican - God bless!
